app/views/posts/show.html.erb

```diff
@@ -23,4 +23,15 @@
         </p>
     <% end %>
     <% end %>
+</div>
+
+<div class='like-comment'>
+
+    <%# add comment %>
+    <%= link_to "Add a comment", new_user_post_comment_path(user_id: @post.author_id, post_id: @post.id), class: "add-comment-btn" %>
+    <%# like %>
+    <%= form_with model: @like, url: user_post_likes_path(post_id: @post.id) do |f| %>
+        <%= f.submit "Like" %>
+    <% end %>
+
```

Comment:
- [ ] [Optional] [Highly recommended] Great job adding 2 buttons(links) to create a new comment and like.
It would be much better professionally if you add a button to add a new post inside the `posts#index` view to allow Users to add Posts. 

![image](https://github.com/umair-rao/New-Blog-App/assets/68796232/73f9de8e-394a-48ad-b83b-9fce1e81a314)